common/src/dev/java/ch/admin/bag/covidcertificate/common/debug/DebugFragment.kt

```diff
@@ -0,0 +1,67 @@
+package ch.admin.bag.covidcertificate.common.debug
+/*
+ * Copyright (c) 2021 Ubique Innovation AG <https://www.ubique.ch>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import ch.admin.bag.covidcertificate.common.databinding.FragmentDebugBinding
+import ch.admin.bag.covidcertificate.sdk.android.net.CertificatePinning
+
+open class DebugFragment : Fragment() {
+
+	companion object {
+		fun newInstance(): DebugFragment = DebugFragment()
+
+		const val EXISTS = true
+
+		fun initDebug(context: Context) {
+			CertificatePinning.enabled = DebugSecureStorage.getInstance(context).isCertPinningEnabled
```

Comment:
ich hab kein enabled frlag auf dem Certifcate Pinning

Reply:
> Marking this as WIP since it depends on admin-ch/CovidCertificate-SDK-Android#64. Then we need to update the SDK here.

The enable flag is introduced in the SDK PR. We need to merge this, release a new SDK version, and then depend on that new version in this branch/PR.